install_miniconda+pyemma.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+# This script installs Miniconda and PyEMMA. If conda is already available, we install PyEMMA in a new environment.
+# By default, we use the name 'pyemma' as environment name.
+# The desired output folders and env names can be set by arguments.
+# Author: Martin Scherer, 2018
+
+function usage {
+    echo "Usage of installation script:"
+    echo "bash $0 -t 'miniconda_target_directory' path to Miniconda"
+    echo "bash $0 -e 'conda_environment' environment name in which pyemma will be installed."
+    echo "bash $0 -i ignore existing miniconda installation. Get a fresh copy."
+    echo "bash $0 -n do not add the new path of miniconda installation to PATH (bashrc update) [off by default]."
+    echo "bash $0 -m minimal set of packags, eg. omit jupyter notebook installation."
+}
+
+# default values for arguments.
+target=$HOME/miniconda3
+env_name="pyemma"
+ignore=1 # dont ignore existing miniconda installation, but just use it to install our environment.
+no_add_bashrc=1 # add new target to PATH in .bashrc
+minimal=1 # minimal set of packags, eg. omit jupyter notebook installation
+
+
+while getopts "ht:e:inm" opt; do
+   case $opt in
+       n) no_add_bashrc=0;;
+       m) minimal=0;;
+       t) target=$OPTARG;;
+       e) env_name=$OPTARG;;
+       i) ignore=0;;
+       h) usage
+         exit 0;;
+       *) echo "unknown option/combination"; exit 0;;
+   esac
+done
+
+
+function add_to_bashrc {
```

Comment:
Doesn't the miniconda installer do this on its own anyway?

Reply:
only if the installer is invoked interactively. We use batch mode and so it does not modify bashrc.